MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -10,36 +10,25 @@ import MapKit
 struct FarmerDetailsViewModel {
 
     let title: String
+    let phoneNumber: String?
+    let productions: [Productions]
```

Comment:
`products` but what actually is `productions`?

Reply:
Yes, in reality these are the products of the farmer so products